user-interface/src/main/java/life/qbic/datamanager/views/projects/project/info/QualityControlList.java

```diff
@@ -0,0 +1,196 @@
+package life.qbic.datamanager.views.projects.project.info;
+
+
+import com.vaadin.flow.component.ClickEvent;
+import com.vaadin.flow.component.ComponentEvent;
+import com.vaadin.flow.component.ComponentEventListener;
+import com.vaadin.flow.component.button.Button;
+import com.vaadin.flow.component.html.Div;
+import com.vaadin.flow.component.html.Span;
+import com.vaadin.flow.component.icon.VaadinIcon;
+import com.vaadin.flow.component.virtuallist.VirtualList;
+import com.vaadin.flow.data.renderer.ComponentRenderer;
+import com.vaadin.flow.shared.Registration;
+import com.vaadin.flow.theme.lumo.LumoIcon;
+import java.util.List;
+import life.qbic.datamanager.views.general.PageArea;
+
+/**
+ * Lists all uploaded {@link QualityControl}. Allows users to upload new {@link QualityControl} and
+ * delete existing {@link QualityControl}.
+ */
+public class QualityControlList extends PageArea {
```

Comment:
The naming of this class is a bit confusing to me, since it is a Component. But maybe we handle other PageAreas in the same way?

Reply:
I think you're absolutely correct 🧠